admin_guide/registry_data_check_restore.adoc

```diff
@@ -0,0 +1,112 @@
+[[admin-guide-registry-data-check-restore]]
+= Image registry data check and restore
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[IMPORTANT]
+====
+All that is described here is an experimental functionality and may
+change in the future.
```

Comment:
"This section describes functionality that is experimental and may change in the future"